### PR TITLE
Adding Flex Cols to fix quiz picture question spacing

### PR DESCRIPTION
**Pull request by nrherron92**
## Description
Modifying grid scss core to add .llms-flex-cols class to utilize flexbox.

## How has this been tested?
I tested it manually (after running npm run build:styles)

## Types of changes
added .llms-flex-cols class
updated multiple choice picture quiz template

## Checklist:
- [x] My code has been tested.
- [ ] My code passes all existing automated tests. 
- [ ] My code follows the LifterLMS Coding & Documentation Standards.

### CODE REVIEW

**Comment by thomasplevy** [comment]
@nrherron92 you based this off dev-600 so switched the base there. That means this will have to wait until 6.0 is released. We can rebase off dev if we feel this needs a release sooner. At this moment I don't have anything else planned for release prior to 6.0 though.

**Comment by nrherron92** [comment]
It can wait (though that wasn't intentional) there's a workaround i can give the user in the meantime! Sorry about that!

File: assets/scss/_includes/_grid.scss [hunk 1]
```diff
@@ -16,8 +16,28 @@
 				width: calc( 100% / $cols );
 			}
 			$cols: $cols + 1;
+		
 		}
-
+
```
Comment (eri-trabiccolo) [inline assets/scss/_includes/_grid.scss:21]:
```suggestion
```
no need to add a new line here.

File: assets/scss/_includes/_grid.scss [hunk 2]
```diff
@@ -16,8 +16,28 @@
...
 	}
 
 }
+
+.llms-flex-cols {
+	display: flex;
+	flex-flow: row wrap;
+
+	[class*="llms-col"] {
+   		flex: 0 1 auto;
```
Comment (eri-trabiccolo) [inline assets/scss/_includes/_grid.scss:31]: This is the initial `flex` value, so we could omit it here, although I'm more inclined to add it as you did, just to be a little bit more sure the cols are not inheriting a different `flex` value from the themes.

**Comment by eri-trabiccolo** [comment]
This is the initial flex value, so we could omit it here, although I'm more inclined to add it as you did, just to be a little bit more sure the cols are not inheriting a different flex value from the themes.
https://github.com/gocodebox/lifterlms/pull/1970/files#diff-590a9c43554f64d68701baa82f21ffa05b8d400e60f61ff447125ad1ae8abf83R31

Can you please delete this line? 
https://github.com/gocodebox/lifterlms/pull/1970/files#diff-590a9c43554f64d68701baa82f21ffa05b8d400e60f61ff447125ad1ae8abf83R21
No need to add a new one there :D

Aside for this it looks great to me! Thanks <3

**Review by eri-trabiccolo — CHANGES_REQUESTED** [review]
https://github.com/gocodebox/lifterlms/pull/1970#issuecomment-1031618957

**Comment by thomasplevy** [comment]
I'm going to rebase this against `dev` so we can ship with the 5.x release I'm prepping.

**Comment by thomasplevy** [comment]
Closing in favor of #1988

**Pull request closed without merging**